app/boards/shields/helix/helix_left.conf

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2020 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+# Central Unit (left)
+CONFIG_ZMK_SPLIT=y
+CONFIG_ZMK_SPLIT_BLE_ROLE_CENTRAL=y
+
+# Enables Sleep functionality (Enabled by default, comment to disable.)
+CONFIG_ZMK_SLEEP=y
+
+# Enables RGB functionality (Uncomment to enable.)
+# CONFIG_ZMK_RGB_UNDERGLOW=y
+# CONFIG_WS2812_STRIP=y
+
+# Enables OLED Function (TO DO. Do not uncomment.)
+# CONFIG_ZMK_DISPLAY=y
```

Comment:
You should remove all of this, as it's unnecessary when the main .conf has the items. The main .conf doesn't work in tree, but it will work in ZMK configs.